baselines/iam-service-test/src/v1/key_management_service_client.ts.baseline

```diff
@@ -0,0 +1,2583 @@
+    ] = gax.routingHeader.fromParams({
+      'parent': request.parent || '',
+    });
+    const callSettings = new gax.CallSettings(options);
+    this.initialize();
+    return this.descriptors.page.listImportJobs.createStream(
+      this.innerApiCalls.listImportJobs as gax.GaxCall,
+      request,
+      callSettings
+    );
+  }
+
+/**
+ * Equivalent to {@link listImportJobs}, but returns an iterable object.
+ *
+ * for-await-of syntax is used with the iterable to recursively get response element on-demand.
+ *
+ * @param {Object} request
+ *   The request object that will be sent.
+ * @param {string} request.parent
+ *   Required. The resource name of the {@link google.cloud.kms.v1.KeyRing|KeyRing} to list, in the format
+ *   `projects/* /locations/* /keyRings/*`.
+ * @param {number} request.pageSize
+ *   Optional limit on the number of {@link google.cloud.kms.v1.ImportJob|ImportJobs} to include in the
+ *   response. Further {@link google.cloud.kms.v1.ImportJob|ImportJobs} can subsequently be obtained by
+ *   including the {@link google.cloud.kms.v1.ListImportJobsResponse.next_page_token|ListImportJobsResponse.next_page_token} in a subsequent
+ *   request. If unspecified, the server will pick an appropriate default.
+ * @param {string} request.pageToken
+ *   Optional pagination token, returned earlier via
+ *   {@link google.cloud.kms.v1.ListImportJobsResponse.next_page_token|ListImportJobsResponse.next_page_token}.
+ * @param {string} request.filter
+ *   Optional. Only include resources that match the filter in the response.
+ * @param {string} request.orderBy
+ *   Optional. Specify how the results should be sorted. If not specified, the
+ *   results will be sorted in the default order.
+ * @param {object} [options]
+ *   Call options. See {@link https://googleapis.dev/nodejs/google-gax/latest/interfaces/CallOptions.html|CallOptions} for more details.
+ * @returns {Object}
+ *   An iterable Object that conforms to @link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols.
+ */
+  listImportJobsAsync(
+      request?: protos.google.cloud.kms.v1.IListImportJobsRequest,
+      options?: gax.CallOptions):
+    AsyncIterable<protos.google.cloud.kms.v1.IImportJob>{
+    request = request || {};
+    options = options || {};
+    options.otherArgs = options.otherArgs || {};
+    options.otherArgs.headers = options.otherArgs.headers || {};
+    options.otherArgs.headers[
+      'x-goog-request-params'
+    ] = gax.routingHeader.fromParams({
+      'parent': request.parent || '',
+    });
+    options = options || {};
+    const callSettings = new gax.CallSettings(options);
+    this.initialize();
+    return this.descriptors.page.listImportJobs.asyncIterate(
+      this.innerApiCalls['listImportJobs'] as GaxCall,
+      request as unknown as RequestType,
+      callSettings
+    ) as AsyncIterable<protos.google.cloud.kms.v1.IImportJob>;
+  }
+/**
+ * This part will be added into src/version/client.ts.
```

Comment:
I don't think this comment should get into the generated code. If it's supposed to be a template comment, use `{#  #}`.

Reply:
sure, moved it!